### ISSUE TITLE
indent not working

### ISSUE THREAD

**Issue by yubaoquan** [buzinas/tslint-eslint-rules#272]
tslint.json
```json
{
    "extends": [
        "tslint-eslint-rules"
    ],
    "rules": {
        "semicolon": [true, "never"],
        "no-extra-semi": true,
        "indent": [true, "spaces", 4],
        "no-consecutive-blank-lines": true,
        "trailing-comma": [true, {
            "multiline": "always",
            "singleline": "never"
        }],
        "space-in-parens": [true, "never"],
        "whitespace": [
            true,
            "check-branch",
            "check-decl",
            "check-operator",
            "check-module",
            "check-separator",
            "check-rest-spread",
            "check-type",
            "check-typecast",
            "check-type-operator",
            "check-preblock"
        ],
        "no-trailing-whitespace": true,
        "one-line": [
            true,
            "check-open-brace",
            "check-else",
            "check-catch",
            "check-finally",
            "check-whitespace"
        ],
        "curly": true,
        "object-curly-spacing": [
            true,
            "always"
        ],
        "one-variable-per-declaration": true,
        "prefer-const": true
    }
}

```

expected
```typescript
function a() {
    const a = 1
    const b = [ 1, 2 ]
    const c = { name: 'x' }
}

```

actual
```typescript
function a() {
 const a = 1
  const b = [ 1, 2 ]
    const c = { name: 'x' }
}

```

**Comment by yubaoquan** [comment]
wrong repo,  see this https://github.com/palantir/tslint/issues/2814
